src/gmail.js

```diff
@@ -338,7 +338,7 @@ var Gmail = function(localJQuery) {
     };
 
     api.dom.inbox_content = function() {
-        return $("div[role=main]:first");
+        return $("div[role=main]:first") && $("[aria-label='Gmail']").length;
```

Comment:
I dont understand this change. Previously the function would return a jQuery element (as formalized in the `gmail.d.ts` typescript definitions), but now you are returning a number?

The documentation also clearly states that this function should return DOM content.

```
DOM

These methods return the DOM data itself

gmail.dom**.inboxes()**
gmail.dom**.inbox_content()**
```

So what's the idea behind this change?